packages/mongo/mongo_driver.js

```diff
@@ -992,6 +992,13 @@ MongoConnection.prototype._createSynchronousCursor = function(
     replaceTypes(cursorDescription.selector, replaceMeteorAtomWithMongo),
     cursorOptions.fields, mongoOptions);
 
+  if (typeof cursorOptions.maxTimeMS !== 'undefined') {
```

Comment:
Was just following Mongo's API, but they're not consistent anyways haha: https://docs.mongodb.com/manual/reference/method/cursor.maxTimeMS/#cursor.maxTimeMS https://docs.mongodb.com/manual/reference/configuration-options/#operationProfiling.slowOpThresholdMs https://docs.mongodb.com/manual/reference/method/db.setProfilingLevel/ which doesn't even capitalize at all ... so that makes sense. Will do.